spec/profile_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'profile'
+
+describe Profile do
+
+  describe '.all' do
+    it 'retrieves the usernames from the database' do
+      profiles = Profile.all
+      expect(profiles).to include('chocolatine')
```

Comment:
Where does chocolatine come from? The test below? Your tests should be independent of each other.